apps/mobile/src/libs/components/text/text.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from "react";
+import { Text as RNText, TextStyle } from "react-native";
+
+import { styles } from "./styles";
+
+type TextProperties = {
+	children: ReactNode;
+	style?: TextStyle | TextStyle[];
+};
+
+const Text = ({ children, style }: TextProperties) => {
```

Comment:
Let's make Text component more generic and add size and preset props. 
Presets for now could be 'default', 'heading' and 'subheading' (if we need, we'll add more later)
You can define sizes (fontSize and lineHeight) based on the Figma (Heading 2 will be the maximum one - we don't need bigger text in the mobile app)

Reply:
Thanks for the review. I've updated the code according to your comment. Could you please take a look?